README.md

```diff
@@ -1,10 +1,10 @@
 # Auth0 Flutter Samples
 
-This is the sample application for the [Auth0 Flutter Quickstart](https://auth0.com/docs/quickstart/native/flutter) using the [auth0_flutter](https://github.com/auth0/auth0_flutter) SDK. The sample source code can be found in the [Sample directory](sample).
+This is the sample application for the [Auth0 Flutter Quickstart](https://auth0.com/docs/quickstart/native/flutter) using the [auth0_flutter](https://github.com/auth0/auth0-flutter/tree/main/auth0_flutter#readme) SDK. The sample source code can be found in the [Sample directory](sample).
 
 ## Issue Reporting
 
-For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums or raise a [support ticket](https://support.auth0.com/). Only [raise an issue](https://github.com/auth0/auth0-flutter-samples/issues) if you have found a bug or want to request a feature.
+For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums or raise a [support ticket](https://support.auth0.com/). Only [raise an issue](https://github.com/auth0-samples/auth0-flutter-samples/issues) if you have found a bug or want to request a feature.
```

Comment:
It looks like the sdks category isn't on the community site anymore, should we instead link to help category or just the landing page?

Reply:
Indeed, good catch! I've updated the link to point to the help category in https://github.com/auth0-samples/auth0-flutter-samples/pull/24/commits/6470a6eda4c1ba4d4c631d0b2472d2cd1a30f810.